src/test/scala/tech/cryptonomic/conseil/tezos/DatabaseConversionsTest.scala

```diff
@@ -819,13 +819,14 @@ class DatabaseConversionsTest
 
       "convert a Ballot operation to a database row" in {
 
-        val converted = (block, groupHash, Ballot: Operation).convertTo[Tables.OperationsRow]
+        val converted = (block, groupHash, sampleBallot: Operation).convertTo[Tables.OperationsRow]
```

Comment:
The test above refers to a type from `TezosTypes` while this this test refers to a local object. Why is this?

Reply:
because ballot now has a parameter, like sampleDelegation/other operations that are case objects instead of case classes in TezosTypes